### DISCUSSION TITLE
Stratospheric assessment of BMT1850 (104) ahead of coupled chemistry simulation

### DISCUSSION THREAD

**Discussion by islasimpson** [NCAR/cesm_dev#11] [2. Model bias]
Here's a look at some features of the stratosphere to inform on whether we are ready to start a BMT1850 with coupled chemistry.

Zonal mean zonal wind.  Looking pretty good.  Maybe a bit too strong in the NH winter.

![zmu](https://github.com/user-attachments/assets/cf5c69f8-331e-4509-9898-7d1be7942bfc)

SH vortex seasonality.  We no longer have the really big problems with the too late breakdown that we had in earlier development runs.  Now in the winter and spring it's looking pretty similar to the CMIP6 WACCM piControl.  Looks a bit better than that in the summer and autumn.

<img width="713" alt="Screen Shot 2024-09-19 at 4 27 29 PM" src="https://github.com/user-attachments/assets/d6ac8a20-ca9f-4289-87ab-67ef010fbac7">

NH vortex seasonality.  Too strong in the late winter and spring.  Maybe we need to try tuning this.

<img width="727" alt="Screen Shot 2024-09-19 at 4 40 05 PM" src="https://github.com/user-attachments/assets/dea9c636-4774-4bff-84bb-d63a88e9ec3f">

Water vapor tape recorder.  

<img width="1206" alt="Screen Shot 2024-09-19 at 5 22 47 PM" src="https://github.com/user-attachments/assets/d61be438-19ed-4dbb-a610-f4afd4769325">

Don't panic.  It's hard to compare 1850 with present day in terms of magnitudes as there are big changes between the periods as shown by the figure from LENS2 below.  There were concerns before that the driest time was happening too late (April/May).  In the above plot it looks like now the driest month is in January i.e., a bit closer to the driest month of February in the observations.

<img width="483" alt="Screen Shot 2024-09-19 at 5 28 56 PM" src="https://github.com/user-attachments/assets/08d3e189-b722-49e2-b8a1-6cd6ca9f9a23">

**Comment by tilmes** [comment]
I ran the MT 104 model run with and without the jet tuning suggested by Julio and here are the diagnostics (thanks to Justin)
https://webext.cgd.ucar.edu/FMTHIST/f.e30_beta02.FMTHIST.ne30.104/atm/f.e30_beta02.FMTHIST.ne30.104_1995_2007_vs_f.e30_beta02.FMTHIST.ne30.104.jet_tuning_1995_2007/index.html 

These also include some seasonal cycle plots, under the WACCM category

**Comment by tilmes** [comment]
Winds in DJF are a little weaker in the jet_tuing run in the NH:
<img width="1020" alt="image" src="https://github.com/user-attachments/assets/9e2f1b2c-b4c8-4afd-94fc-f49401fcd48c">

and temperatures are warmer in Nov/Dec but better in spring:
<img width="784" alt="image" src="https://github.com/user-attachments/assets/57b9336a-fed9-4574-8df5-300ab8322fa8">


and a little weaker in JJA in the SH:
<img width="994" alt="image" src="https://github.com/user-attachments/assets/0b76c988-9963-4867-8cde-3b090e8742a7">

temperatures are improved in Sep/Oct:
<img width="783" alt="image" src="https://github.com/user-attachments/assets/f2e4075d-b979-4f90-801a-784b6299d28e">

**Comment by tilmes** [comment]
And here is the tape recorder:
<img width="809" alt="image" src="https://github.com/user-attachments/assets/9d0a12af-7042-488c-8f22-5cc53eb18943">

**Comment by JulioTBacmeister** [comment]
Hi Simone which run is the one labeled 'cool nickname'?

**Reply by tilmes** [reply]
the one without the jet tuning :)

**Reply by JulioTBacmeister** [reply]
Thanks!

**Reply by JulioTBacmeister** [reply]
So do we have the patience to try another run with just the moving mountains strngthened?

**Reply by tilmes** [reply]
I don't mind :) Can you add the namelist changes, so I know what to do?

**Reply by islasimpson** [reply]
Are there any plots that are still needed?  I had it on my list to look at these runs but haven't got to it yet.  It seems like most of the diagnostics are covered with the ADF?
